### PR TITLE
Added new templates.

### PR DESCRIPTION
**Pull request by jhamilton1**
**What this PR does / why we need it**:
The PR adds standardized bug and feature-enhancement templates. Starting with sonobuoy we will apply these to all repos.

**Which issue(s) this PR fixes**
- Fixes #
N/A

**Special notes for your reviewer**:
I will start a document to track which repos have been updated with these templates.

**Release note**:
None
```
release-note
None
```

### CODE REVIEW

File: .github/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+**Anything else you would like to add:**
+[Miscellaneous information that will assist in solving the issue.]
+
+
+**Environment:**
+
+- Sonobuoy version: 
+- Kubernetes version: (use `kubectl version`):
+- Kubernetes installer & version:
```
Comment (chuckha) [inline .github/bug_report.md]: maybe this could be: Kubernetes installer & installer version (e.g. kops, kubeadm, kubespray)

**Comment by chuckha** [comment]
Oh, @jhamilton1 you will have to sign off your commit with git. Also please squash down to 1 commit to get a nice commit message otherwise we'll squash this side and all the commit messages will be concatenated and end up in one big commit message.

Thank you for working on this!

**Comment by jhamilton1** [comment]
Okay, I will get that straightened out.

**Comment by jhamilton1** [comment]
Okay, @chuckha  I squashed down to 1 commit and signed that off.  Looks like I am still failing that check though.

**Comment by timothysc** [comment]
@jhamilton1 - nope now you have 5 commits.

**Pull request merged**